packages/fxa-payments-server/src/routes/Subscriptions/index.scss

```diff
@@ -250,6 +249,7 @@
 .pocket-external {
   display: flex;
   align-items: baseline;
+  gap:.75rem;
```

Comment:
```suggestion
  gap: 0.75rem;
```

Reply:
Oops, sorry! Thanks for catching, @pdehaan!

Reply:
Thanks @pdehaan! Fixed in latest push